library/core/src/iter/adapters/enumerate.rs

```diff
@@ -23,6 +23,10 @@ impl<I> Enumerate<I> {
     pub(in crate::iter) fn new(iter: I) -> Enumerate<I> {
         Enumerate { iter, count: 0 }
     }
+
+    pub fn peek_index(&self) -> usize {
```

Comment:
This needs to be annotated with the `#[unstable]` attribute, like so
```suggestion
    #[unstable(feature = "feature_name", issue = "some issue's number")]
    pub fn peek_index(&self) -> usize {
```

Reply:
And yes, this does indeed imply you have to create a tracking issue for it which can be linked to.

Reply:
I will be able to add this tomorrow, gitgub mobile is really unusable